src/components/experiences/social-login/index.js

```diff
@@ -26,30 +27,27 @@ function SocialLogin(props) {
         locale,
     } = props;
     return (
-        <article className="rainbow-auth-firebase-social-login_buttons-container">
+        <StyledArticle>
             <div>
-                <Button
-                    className="rainbow-auth-firebase-social-login_button rainbow-auth-firebase-social-login_facebook-button"
+                <StyledFacebookButton
                     onClick={loginWithFacebook}
                     isLoading={isLoadingFacebook}>
-                    <FacebookIcon className="rainbow-auth-firebase-social-login_social-icon" />
+                    <StyledFacebookIcon />
                     <FormattedMessage id="facebook.button.login" defaultMessage="Login with Facebook" />
-                </Button>
-                <Button
+                </StyledFacebookButton>
+                <StyledLoginButton
                     variant="neutral"
-                    className="rainbow-auth-firebase-social-login_button"
                     onClick={loginWithGoogle}
                     isLoading={isLoadingGoogle}>
-                    <GoogleIcon className="rainbow-auth-firebase-social-login_social-icon" />
+                    <StyledGoogleIcon />
```

Comment:
I think that you don't need to create a `StyledGoogleIcon` since it and the `StyledFacebookIcon` have the same styles.
You can create a `StyledIcon` with the `FacebookIcon` and after pass `as={GoogleIcon}`